Event/FilterLocaleSwitchEvent.php

```diff
@@ -36,7 +36,7 @@ class FilterLocaleSwitchEvent extends Event
      */
     public function __construct(Request $request, $locale)
     {
-        if (!is_string($locale) || null == $locale || '' == $locale) {
+        if (!is_string($locale)) {
```

Comment:
i think this is incorrect. you need to keep the check for empty string, as that would be `is_string` but still not ok.